backend/fms_core/migrations/0031_v3_7_0.py

```diff
@@ -0,0 +1,143 @@
+                ('created_by', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.PROTECT, related_name='fms_core_indexstructure_creation', to=settings.AUTH_USER_MODEL)),
+                ('flanker_3prime_forward', models.ForeignKey(help_text="Flanker on the 3' forward direction", on_delete=django.db.models.deletion.PROTECT, related_name='flanker_3prime_forward', to='fms_core.sequence')),
+                ('flanker_3prime_reverse', models.ForeignKey(help_text="Flanker on the 3' reverse direction", on_delete=django.db.models.deletion.PROTECT, related_name='flanker_3prime_reverse', to='fms_core.sequence')),
+                ('flanker_5prime_forward', models.ForeignKey(help_text="Flanker on the 5' forward direction", on_delete=django.db.models.deletion.PROTECT, related_name='flanker_5prime_forward', to='fms_core.sequence')),
+                ('flanker_5prime_reverse', models.ForeignKey(help_text="Flanker on the 5' reverse direction", on_delete=django.db.models.deletion.PROTECT, related_name='flanker_5prime_reverse', to='fms_core.sequence')),
+                ('updated_by', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.PROTECT, related_name='fms_core_indexstructure_modification', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='IndexSet',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, help_text='Date the instance was created.')),
+                ('updated_at', models.DateTimeField(auto_now=True, help_text='Date the instance was modified.')),
+                ('deleted', models.BooleanField(default=False, help_text='Whether this instance has been deleted.')),
+                ('name', models.CharField(help_text='The name of the index set.', max_length=200, unique=True, validators=[django.core.validators.RegexValidator(re.compile('^[a-zA-Z0-9.\\-_]{1,200}$'))])),
+                ('created_by', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.PROTECT, related_name='fms_core_indexset_creation', to=settings.AUTH_USER_MODEL)),
+                ('updated_by', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.PROTECT, related_name='fms_core_indexset_modification', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.AddField(
+            model_name='index',
+            name='index_set',
+            field=models.ForeignKey(help_text='The set which this index belongs to', on_delete=django.db.models.deletion.PROTECT, related_name='indeces', to='fms_core.indexset'),
+        ),
+        migrations.AddField(
```

Comment:
As much as possible, do not add model fields apart from model creation. Makes tracking if everything is present harder.